utils/oc.py

```diff
@@ -233,6 +239,11 @@ def get_api_resources(self):
         results = self._run(cmd).decode('utf-8').split('\n')
         return [r.split()[-1] for r in results]
 
+    def get_cluster_info(self):
+        # get cluster-info to check that the cluster is reachable
```

Comment:
this comment is a bit misleading.... it makes sense in the line 73 aboce, but it doesn't here... here you're simply calling cluster-info... this method knows nothing about your use case for that, if you know what I mean.

Reply:
agreed will change